src/styles/cards.scss

```diff
@@ -0,0 +1,84 @@
+.cards {
+  background-color: $white-color;
+  width: $card-size;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $gray-color;
+  padding: 32px 16px 16px;
+  justify-self: center;
+  border-radius: 5px;
+  transition: all 0.3s;
```

Comment:
Not critical but it's better to specify the properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x